src/coverlet.core/Coverage.cs

```diff
@@ -386,6 +376,16 @@ private void CalculateCoverage()
           }
         }
 
+        if (!_fileSystem.Exists(result.HitsFilePath))
```

Comment:
Can you elaborate this change?

Reply:
The issue #1679 describes the problem and changing the sequence make sure the file name for activated SourceLink option is **always** updated.